### DISCUSSION TITLE
Null safety issues with one of your flutter plugins

### DISCUSSION THREAD

**Discussion by Ferdinand77Raw** [community/community#43986] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Error: Cannot run with sound null safety, because the following dependencies
don't support null safety:

 - package:sip_native
 
 There's this issue with your plugin, I don't know if it is outdated or what, I tried unsound null safety and did not work either. I think you should upgrade your page so that in a future this package can work properly.

**Comment by bzkarim250** [comment]
you can use this guide [Understand unsound null safety](https://dart.dev/null-safety/unsound-null-safety) ,
 then run this command :  flutter run --no-sound-null-safety  , . fater being sure that you want to run with unsound null safety
